tools/wasm_demo/service_worker.js

```diff
@@ -182,6 +184,8 @@
   };
 
   const serviceWorkerMain = () => {
+    leak = WebAssembly.compileStreaming(fetch("jxl_decoder.wasm"));
```

Comment:
Why not build with streaming compile through emscripten rather than trying to add it as a side-load? I suspect it only works some of the time since it's likely racing emscripten's own wasm loader.
https://emsettings.surma.technology/#MINIMAL_RUNTIME_STREAMING_WASM_COMPILATION

Reply:
As you can see, variable is named `leak` - we stream-compile decoder in the context of service-worker and never use; that is done only to enable compiled WASM code caching.